### PR TITLE
Create contractor for swl and bl update. Get document info for LicenceResponse

### PR DESCRIPTION
**Pull request by peggy-quartech**
# Description

This PR includes the following proposed change(s):

- Create contractor for swl and bl update. 
- Get document info for LicenceResponse

### CODE REVIEW

File: src/Spd.Resource.Repository/Document/DocumentRepository.cs [hunk 1]
```diff
@@ -40,7 +40,10 @@ public async Task<DocumentListResp> QueryAsync(DocumentQry qry, CancellationToke
     {
         var documents = _context.bcgov_documenturls.Where(d => d.statecode != DynamicsConstants.StateCode_Inactive);
         if (qry.ApplicantId != null)
-            documents = documents.Where(d => d._spd_submittedbyid_value == qry.ApplicantId);
+            documents = documents.Where(d => d._bcgov_customer_value == qry.ApplicantId);
```
Comment (esdd1995) [inline src/Spd.Resource.Repository/Document/DocumentRepository.cs:43]: do we know this change would effect on other apps that have reference to this function?
there is 46 refs

File: src/Spd.Manager.Licence/SecurityWorkerLicenceManager.cs [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    /// </summary>
+    /// <param name="cmd"></param>
+    /// <param name="cancellationToken"></param>
+    /// <returns></returns>
+    /// <exception cref="ArgumentException"></exception>
+    public async Task<WorkerLicenceCommandResponse> Handle(WorkerLicenceUpdateCommand cmd, CancellationToken cancellationToken)
+    {
+        return null;
+
+    }
```
Comment (esdd1995) [inline src/Spd.Manager.Licence/SecurityWorkerLicenceManager.cs:90]: wow looks like my future jira board😄

**Review by esdd1995 — APPROVED** [review]

File: src/Spd.Resource.Repository/Document/DocumentRepository.cs (hunk 1 above)
Comment (peggy-quartech) [reply src/Spd.Resource.Repository/Document/DocumentRepository.cs:43]: yeah, actually, the old code is wrong.

File: src/Spd.Manager.Licence/SecurityWorkerLicenceManager.cs (hunk 2 above)
Comment (peggy-quartech) [reply src/Spd.Manager.Licence/SecurityWorkerLicenceManager.cs:90]: you bet.

**Pull request merged**